src/main/java/seedu/address/logic/commands/GitHubCommand.java

```diff
@@ -0,0 +1,89 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.io.IOException;
+
+import seedu.address.commons.core.Browser;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Github;
+import seedu.address.model.person.Name;
+
+/**
+ * Launches a person's GitHub repository from the address book.
+ */
+public class GitHubCommand extends Command {
+    public static final String COMMAND_WORD = "github";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Launches the github repository of the user. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "John Doe ";
+    public static final String MISSING_PERSON_EXCEPTION = "The person you specified does not exist"
+            + " in the address book."
+            + "Parameters: "
```

Comment:
I think there should be a new line or blank space between `.` and `Parameters`.

Reply:
Yup I agree!